### ISSUE TITLE
Unknown construction=* values hide construction roads

### ISSUE THREAD

**Issue by imagico** [gravitystorm/openstreetmap-carto#4725]
Similar to #4568 we have another weird catch-all interpretation of tags in display of construction roads.  When a `highway=construction` is tagged with a construction value we do not explicitly interpret like `construction=foo` we do not render them - while we do render them if there is no `construction=*` tagged at all:

![construction](https://imagico.de/files/carto_construction.png)

Reason is this:

https://github.com/gravitystorm/openstreetmap-carto/blob/4d1b5ff99669f2172b267f7231f27fcf652e6b95/style/roads.mss#L1227

As a general principle we tend to ignore unknown tags, i.e. `construction=foo` should be interpreted as if no `construction=*` is tagged to allow mappers the freedom to invent new tags without that being punished by us interpreting it as explicitly invalid.  Even if we'd want to enforce a limited set of highway types the resulting feedback through hiding is confusing.

**Comment by matkoniecz** [comment]
another option is hiding bare `highway=construction` (see #4723)

It is also confusing when `highway=construction construction=ant_tunnel` will be rendered and `highway=ant_tunnel` will not be (example hopefully fictional)

**Comment by imagico** [comment]
Yes, that would be an option.  It would however incentivize adding a made up/guessed `construction=*` value for it to show up in cases where the type of road is - at the construction stage - not possible to determine for the mapper. 221k of 304k `highway=construction` ways have a `construction=*` tag.

**Comment by mvglasow** [comment]
If the type of road cannot be determined during construction, I would suggest tagging the road `highway=construction, construction=road`. Similar to how `highway=road` is used for a road of an unknown type, and consistent with how we tag other road types under construction. This hopefully gets rendered (if not, it should).
